docs/api/markers.rst

```diff
@@ -4,3 +4,11 @@ Markers
 .. automodule:: junifer.markers
    :members:
    :imported-members:
+
+
+Complexity
+----------
+
+.. automodule:: junifer.markers.complexity
```

Comment:
Will doc build without the neurokit2 dependency? Just asking, I don't know.

Reply:
I've added it as a dependency for building docs preview, once that is ok, will add it for building docs.

Reply:
Done.